Makefile

```diff
@@ -68,18 +68,22 @@ check-install: ## Checks that you've installed this repository correctly
 .PHONY: manager
 manager: generate  ## Build manager binary.
 	bazel build //cmd/manager $(BAZEL_ARGS)
+	cp -f bazel-bin/cmd/manager/*/manager $(shell go env GOPATH)/bin/aws-manager
```

Comment:
Using `install` might be better here since the permissions of the bazel built binary lacks write permissions.

Reply:
i do like `install` more than `cp -f`.